cli.js

```diff
@@ -6,6 +6,27 @@ const chalk = require('chalk');
 const ora = require('ora');
 const { api, utils } = require('dext-core-utils');
 
+args.command(['search', 's'], 'Seach for plugins or themes.', (name, sub) => {
+  const searchTerm = sub[0];
+  const spinner = ora(chalk.green(`Searching for "${searchTerm}"...`)).start();
+  return api.search(searchTerm)
+    .then(results => {
+      spinner.color = 'green';
+      spinner.text = chalk.green(`Results for "${searchTerm}":`);
+      spinner.succeed();
+      if(results.length > 0) {
+        results.forEach(result => console.log(' - ' + result));
+      } else {
+        console.log(chalk.yellow('no packages found'));
+      }
```

Comment:
There are cleaner approach to this.

First: use reduce or map to build a list of strings. At least in the end, you can `console.log` __once__
Second: use string interpolation/templating coming with ES6

just an example

```js
const resultMessage = results.length
  ? results.reduce(result => `- ${result}`, '').join('\n')
  : 'no packages found'

console.log(chalk.yellow(resultMessage))
```